### DISCUSSION TITLE
How to work conversions between jupyter and org

### DISCUSSION THREAD

**Discussion by caio-a-garcia** [jgm/pandoc#9434] [Q&A]
Hello! I have an upcoming project where my colaborators will all be working with Jupyter, and I was wondering if pandoc would allow me to colaborate over an org file. In my specific case, this means:
 - I need conversion to work both ways (ipynb->org to get updates, org->ipynb to submit my changes)
 - I'd rather not have pandoc-specific text in the ipynb file (but a separate config file would be fine)

So far I tried converting a sample ipynb file to org and both my code and outputs were converted to `example` blocks, where I'd expect the code to be converted to `src` blocks. I can think of workarounds to this, but would like to know if I just did something wrong on the process.

In my research so far I have looked at jupytex (which [led me here](https://github.com/mwouts/jupytext/issues/62)) and noticed [ox-ipynb](https://github.com/jkitchin/ox-ipynb) which I haven't looked much into, but would seem to address only half of what I need. Searching pandoc's documentation and this forum I haven't noticed anything that seemed directly relevant.

What I hope to find here:
 - Documentation or previous discussions relevant to what I'm trying to do (e.g. directions to customize pandoc's behaviour localy, if what I'm trying to do is considered very specific)
 - Some direct solution to my problem, in case someone has done this before
 - or A better understanding of the situation, in case all this is not practical with pandoc

Thank you for your time and attention :)

**Comment by jgm** [comment]
Pandoc ought to produce src blocks normally...can you give an example of one of the cells that is not being converted that way (or even better a complete ipynb)?

**Reply by caio-a-garcia** [reply]
I just added the test I made to a [repo](https://github.com/caio-a-garcia/pandoc-issue-example-1). In this case, the ipynb is the original, while the org file was generated by `pandoc -o Analise_Exploratoria.org Analise_Exploratoria.ipynb`.

**Comment by caio-a-garcia** [comment]
Also just realized I am using version 2.19, which is the version packaged in guix (in the gnu channel, if I'm not mistaken). If this is the issue, I guess my original point can be considered resolved, in which case any direction on how to get a more up to date version of pandoc on guix will be much appreciated.

**Comment by jgm** [comment]
You can test it with the latest pandoc at https://pandoc.org/try/

**Reply by caio-a-garcia** [reply]
It worked, the issue was indeed the version. Thank you!

**Comment by caio-a-garcia** [comment]
I'd like to ask, in case anyone knows, if there is some guix channel where I can get a (mostly) up-to-date version of pandoc. I'll mark this topic as closed though, since the answer to my original question is that **pandoc works as expected on the latest version**.
